tests/shell/main.c

```diff
@@ -121,8 +145,8 @@ int main(void)
 
     /* or use only system shell commands */
     /*
-    shell_run(NULL, line_buf, SHELL_DEFAULT_BUFSIZE);
-    */
+       shell_run(NULL, line_buf, SHELL_DEFAULT_BUFSIZE);
+     */
```

Comment:
Nit-pick: [`vim-commentary`](https://github.com/tpope/vim-commentary) doesn't like either format. The previous format at least allowed for only removing the `/*` and `*/` to have clean code. An `uncrustrify`-compliant way that `vim-commentary` would also be happy with could be

```C
    /* shell_run(NULL, line_buf, SHELL_DEFAULT_BUFSIZE); */
```

Reply:
fixed